tests/network-tests/src/fixtures/membership/UpdateProfileHappyCaseFixture.ts

```diff
@@ -93,6 +97,7 @@ export class UpdateProfileHappyCaseFixture extends BaseQueryNodeFixture {
       expected.externalResources?.map(asMembershipExternalResource) ?? []
     )
     assert.isFalse(Utils.hasDuplicates(newMetadata.externalResources?.map(({ type }) => type)))
+    assert.equal(newMetadata.isVerifiedValidator, false)
```

Comment:
If you remove `assert.equal(newMetadata.validatorAccount, expected.validatorAccount)` you're not actually checking if the `validatorAccount` was added to the member metadata in the QN. `validatorAccount` shouldn't be blank. It should be 'validator address' after the first update, then 'validator address 2', then it should remain 'validator address 2' and finaly it should be `null` after the 4th update. If it's not the case something is wrong either with the test or the mapping.